### DISCUSSION TITLE
Question about simulator for other agents

### DISCUSSION THREAD

**Discussion by JunyongYun-SPA** [waymo-research/waymax#86] [Q&A]
Hi, I am interested in your simulator for planning project.

I have a question about your simulator for other agents.

I understood that Waymax provides various simulators such as a rule-based IDM, and log play. And I know that Waymax allows us to use any model for the simulator. 

So, Can you provide other models such as models learned via imitation learning except for IDM and log-play for the simulator?

Thank you.

**Comment by samuela** [comment]
Yes, you can specify an arbitrary policy function to be used for each agent. See eg https://github.com/waymo-research/waymax/blob/main/docs/notebooks/multi_actors_demo.ipynb for example code!
